package.json

```diff
@@ -107,11 +109,13 @@
     "@types/glob": "^8.0.0",
     "@types/node": "^18.11.9",
     "@types/stack-utils": "^2.0.1",
+    "@types/vscode": "^1.73.0",
     "@types/which": "^2.0.1",
     "@types/ws": "^8.5.3",
     "@typescript-eslint/eslint-plugin": "^5.44.0",
     "@typescript-eslint/parser": "^5.44.0",
-    "esbuild": "^0.15.15",
+    "@vscode/l10n-dev": "^0.0.24",
+    "esbuild": "^0.17.17",
```

Comment:
I believe the esbuild version bump was not related.

Reply:
sure. will rollback to ^0.15.15